frontend/services/run.js

```diff
@@ -0,0 +1,12 @@
+import store from '../state/stores/runStore';
+
+class RunService {
+    async fetchData(id) {
```

Comment:
```suggestion
    async fetchOne(id) {
```

Could add another method `fetchList` and merge with `RunStoreService`.

Reply:
I thinks let's keep them separate for extensibility?

Reply:
I would argue that since they are operating on the same noun (`run`), they could be kept in a single service. If we ever find that they need to be in a separate service (which I doubt 😅), we can move to separate service. Thoughts?